README.md

```diff
@@ -1,16 +1,15 @@
-<div align="center">
+# IBM Generative AI CLI (Tech Preview)
 
-# Generative AI CLI
+This is not the [watsonx.ai](https://www.ibm.com/products/watsonx-ai) CLI. This is the CLI for the Tech Preview program for IBM Foundation Models Studio.
+You can start a trial version or request a demo via https://www.ibm.com/products/watsonx-ai.
```

Comment:
It's misleading to have the info about the trial here. Reader would expect it references a Tech Preview trial.
```suggestion
```